include/NeoFOAM/dsl/solver.hpp

```diff
@@ -43,8 +45,10 @@ void solve(
     if (exp.temporalOperators().size() > 0)
     {
         // integrate equations in time
-        TimeIntegration<FieldType> timeIntegrator(fvSchemes.subDict("ddtSchemes"));
-        timeIntegrator.solve(exp, solution, dt);
+        NeoFOAM::timeIntegration::TimeIntegration<FieldType> timeIntegrator(
```

Comment:
Since we are in NeoFOAM namespace can this be shortened to
```suggestion
        timeIntegration::TimeIntegration<FieldType> timeIntegrator(
```
?